### PR TITLE
Create turnoff.xml

### PR DESCRIPTION
**Pull request by Tea23**
turnoff.us is a weird Linux-themed comic thing

### CODE REVIEW

**Comment by Roman3349** [comment]
Please you use tabs not spaces. And remove redirect from www.turnoff.us to turnoff.us.

File: src/chrome/content/rules/turnoff.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<ruleset name="Turnoff">
+        <target host="turnoff.us" />
+        <target host="www.turnoff.us" />
+
+        <rule from="^http:"
+                to="https:" />
+
+  <rule from="^https://(?:www\.)?turnoff\.us/"
+          to="https://turnoff.us/" />
```
Comment (J0WI) [inline src/chrome/content/rules/turnoff.xml]: This block is not needed. Both targets are working with https.

**Comment by fuglede** [comment]
Added the suggested change to #4746, simply to up the backlog a bit.

**Comment by J0WI** [comment]
Closing in favor of #4746

**Pull request closed without merging**